上周完成了一个 python 脚本，可以自动帮我做不少数据的统计分析。已经成功运行过。今天运行时，遇到一个报错：

> urllib.error.URLError: <urlopen error unknown url type: https>

我尝试了重装 urllib，也尝试了把 url 中的 : 改成 %3A ……网上推荐的做法，逐一尝试，就是不成功。

发现一个回答是：需要 openssl 模块。或者 openssl 没成功。无意中又看到：“win10 Anaconda python 中 SSL 模块无法正常使用”……

我昨晚在已直接装过 vscode 的情况下，又通过 anaconda 装过 vscode，这中间可能发生了什么……

于是添加几条环境变量……问题解决。脚本可正常运转了。说起来简单，这过程中绕了很多圈儿……
